packages/core/upload/admin/src/components/UploadAssetDialog/PendingAssetStep/PendingAssetStep.js

```diff
@@ -30,15 +31,36 @@ export const PendingAssetStep = ({
   onClickAddAsset,
   onCancelUpload,
   onUploadSucceed,
+  trackedLocation,
 }) => {
   const assetCountRef = useRef(0);
   const { formatMessage } = useIntl();
+  const { trackUsage } = useTracking();
   const [uploadStatus, setUploadStatus] = useState(Status.Idle);
 
   const handleSubmit = async e => {
     e.preventDefault();
     e.stopPropagation();
 
+    const assetsCountByType = assets.reduce((acc, asset) => {
+      const { type } = asset;
+
+      if (!acc[type]) {
+        acc[type] = 0;
+      }
+
+      // values need to be stringified because Amplitude ignores number values
+      acc[type] = parseInt(acc[type], 10) + 1;
+      acc[type] = acc[type].toString();
```

Comment:
What do you think about:

```suggestion
      acc[type] = `${parseInt(acc[type], 10) + 1}`;
```

I think this is nicer to read :)

Reply:
oh didn't think about that I like it better, thanks!